src/main/scala/mesosphere/marathon/core/groupmanager/GroupManagerConfig.scala

```diff
@@ -0,0 +1,13 @@
+package mesosphere.marathon.core.groupmanager
+
+import org.rogach.scallop.ScallopConf
+
+trait GroupManagerConfig extends ScallopConf {
+  //scalastyle:off magic.number
+
+  lazy val groupManagerRequestTimeout = opt[Int](
+    "group_manager_request_timeout",
```

Comment:
All command line flags are configured in ms and don't have the unit in the name. But the description should have the unit.